realm/realm-library/src/main/java/io/realm/RealmNativeSchema.java

```diff
@@ -0,0 +1,127 @@
+/*
+ * Copyright 2015 Realm Inc.
```

Comment:
Yeah... I had guessed at that, but I wasn't positive.  It looked to me as if the code that ended up in `RealmNativeSchema didn't support DynamicRealm much.  Glad to get the confirmation.  How should it work?  How should it work until `ObjectStoreSchema` supports `DynamicRealm`?

Fixed the CR date.